### DISCUSSION TITLE
Behaviors in other plugins are broken

### DISCUSSION THREAD

**Discussion by joshuabaker** [solspace/craft-freeform#661] [Feature Requests]
At some point between [v3.13.18 and v3.13.21](https://github.com/solspace/craft-freeform/compare/v3.13.18...v3.13.21) a fairly hidden bug was introduced that breaks Craft’s behaviors flow in other plugins.

Reverting and retaining the install at v3.13.18 seems to avoid these issues.

The downstream issue is noted here: studioespresso/craft-scout#257

It’s possible [this comment](https://github.com/studioespresso/craft-scout/pull/105#issuecomment-533157786) on a related Scout issue will help.

**Craft & Plugin Info**

- Craft Version: 3.7.68
- Freeform Edition: Pro

**Comment by kjmartens** [comment]
Thanks for reporting this @joshuabaker. We'll check into it. 🙂

**Comment by kjmartens** [comment]
Hi again @joshuabaker,

I had a developer check into this and we don’t see any place where we ever touch `searchable` and the link in `v3` of Freeform repo provided is a dead end, and subsequently doesn’t show what causes the issue.

Looking back, I can see we addressed some kind of issue like this before:
<img width="1785" alt="Screenshot 2023-03-22 at 12 18 21 AM" src="https://user-images.githubusercontent.com/1821674/226808828-f832209b-1ff7-47ab-a1f0-a0d919e87e44.png">

But we're not sure what needs to be changed... do you have any other information you might be able to provide about this one? 🙂
